last-module-codecoverage-check/pom.xml

```diff
@@ -112,11 +112,6 @@
             <version>${project.version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>de.adorsys</groupId>
-            <artifactId>datasafe-simple-adapter-api</artifactId>
-            <version>${project.version}</version>
-        </dependency>
```

Comment:
All datasafe modules should be listed in last-module-codecoverage-check's pom.xml. It's needed for jacoco to aggregate reports.